plotgui.py

```diff
@@ -295,32 +329,73 @@ def contextMenuEvent(self, event):
 
         self.menu.exec_(event.globalPos())
 
-    def setPixmap(self, w, h):
+    def setPixmap(self, w=None, h=None):
 
         # clear out figure
         self.figure.clear()
 
         cv = self.model.currentView
         # set figure bg color to match window
-        window_background = self.parent.palette().color(QtGui.QPalette.Background)
-        self.figure.patch.set_facecolor(rgb_normalize(window_background.getRgb()))
+        window_bg = self.parent.palette().color(QtGui.QPalette.Background)
+        self.figure.patch.set_facecolor(rgb_normalize(window_bg.getRgb()))
+
         # set figure width
-        self.figure.set_figwidth(0.99 * w / self.figure.get_dpi())
-        self.figure.set_figheight(0.99 * h / self.figure.get_dpi())
+        if w:
+            self.figure.set_figwidth(0.99 * w / self.figure.get_dpi())
+        if h:
+            self.figure.set_figheight(0.99 * h / self.figure.get_dpi())
         # set data extents for automatic reporting of pointer location
         data_bounds = [cv.origin[self.mw.xBasis] - cv.width/2.,
                        cv.origin[self.mw.xBasis] + cv.width/2.,
                        cv.origin[self.mw.yBasis] - cv.height/2.,
                        cv.origin[self.mw.yBasis] + cv.height/2.]
 
         # make sure we have an image to load
-        if not hasattr(self.model,'image'):
+        if not hasattr(self.model, 'image'):
             self.model.generatePlot()
-        c = self.figure.subplots().imshow(self.model.image,
-                                          extent=data_bounds,
-                                          alpha=cv.plotAlpha)
-        self.ax = self.figure.axes[0]
 
+        if cv.colorby in ('material', 'cell'):
+            self.image = self.figure.subplots().imshow(self.model.image,
+                                                       extent=data_bounds,
+                                                       alpha=cv.plotAlpha)
+        else:
+            cmap = cv.colormaps[cv.colorby]
+            if cv.colorby == 'temperature':
+                idx = 0
+                cmap_label = "Temperature (K)"
+            else:
+                idx = 1
+                cmap_label = "Density (g/ccm)"
```

Comment:
g/cc?

Reply:
That's a pretty rough typo. Thanks for catching that.